02_repeatString/repeatString.js

```diff
@@ -1,6 +1,10 @@
-const repeatString = function() {
-
+const repeatString = function (string, number) {
+  if (number < 0) return "ERROR";
+  let answer = "";
+  for (let i = 0; i < number; i++) {
+    answer = answer + string;
+  }
+  return answer;
```

Comment:
Spacing always helps readability:
```suggestion
  if (number < 0) return "ERROR";
  
  let answer = "";
  
  for (let i = 0; i < number; i++) {
    answer = answer + string;
  }
  
  return answer;
```